container-setup/install/install-utils.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+if [ "$I_AM_IN_CONTAINER" != "I-am-in-container" ]; then
+  echo "must be run in container";
+  exit 1;
+fi
+
+echo "in container";
+
+mkdir -p ${HOME}/utils
+mkdir -p /etc/profile.d
+echo "export PATH=${PATH}:${HOME}/utils" > /etc/profile.d/localbin.sh
```

Comment:
If this is running inside a container, designed to be run as root anyway, why not just put the utilities in one of the normal $PATH bin directories?  Or use ~/.local/bin rather than a custom dir?  This file could be replaced with one line in the Dockerfile, maybe.

Reply:
Funny you mention this, I actually had to make some edits to this to get `docker exec` to work and auto-login as well as to be able to run commands from outside the container.

I ended up changing the path in the Dockerfile and removing this in another PR that will come up here after this one merges.